Sources/Datadog/DatadogConfiguration.swift

```diff
@@ -723,8 +719,9 @@ extension Datadog {
             ///     SWIFT_ACTIVE_COMPILATION_CONDITIONS = DD_SDK_ENABLE_INTERNAL_MONITORING
             ///
             /// - Parameter clientToken: the client token authorised for a Datadog org which should receive the SDK telemetry
+            @available(*, deprecated, message: "Internal monitoring has been removed")
             public func enableInternalMonitoring(clientToken: String) -> Builder {
-                configuration.internalMonitoringClientToken = clientToken
+                // no-op
```

Comment:
We only use this API in dogfooding, so we can remove it entirely, along with its surrounding compiler condition. When upgrading `dd-sdk-ios` in our projects, we will just make the chore of removing it, WDYT?